generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -0,0 +1,1532 @@
+# Do not edit this file; it was automatically generated.
+
+import grpc
+import threading
+import warnings
+
+from . import errors as errors
+from nidaqmx._base_interpreter import BaseInterpreter
+from nidaqmx._stubs import nidaqmx_pb2 as grpc_types
+from nidaqmx._stubs import nidaqmx_pb2_grpc as nidaqmx_grpc
+from nidaqmx._stubs import session_pb2 as session_grpc_types
+
+
+
+class GrpcStubInterpreter(BaseInterpreter):
+    '''Interpreter for interacting with a gRPC Stub class'''
+
+    def __init__(self, grpc_options):
+        self._grpc_options = grpc_options
+        self._lock = threading.RLock()
+        self._client = nidaqmx_grpc.NiDAQmxStub(grpc_options.grpc_channel)
```

Comment:
The `GrpcSessionOptions` class is not defined yet.

Copy https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/grpc_session_options.py into `src/handwritten/` and update it for DAQmx.

Reply:
This is covered as part of another PR. https://github.com/ni/nidaqmx-python/pull/288